### ISSUE TITLE
Support new element for fine tunning

### ISSUE THREAD

**Issue by sandipgiri576** [Fung-Lab/MatterTune#18]
Hi! Nice repo. 
Is it possible to fine-tune a pretrained foundation model (for example, MACE-OFF, which supports C, H, N, O) with a new training dataset that is completely new and has a different set of elements (for example, B, Cl, etc)?

**Comment by Lingyu-Kong** [comment]
Hi, the answer to this question may depend on the type of foundation model. We believe that for most foundation models, fine-tuning on datasets outside the pretraining domain is feasible, although the process may be somewhat unstable and could lead to a loss of certain pretrained knowledge. For a few foundation models, however, there may be issues with mismatched atomic encodings.
